elements/rh-card/rh-card.css

```diff
@@ -10,12 +10,13 @@ article {
 }
 
 ::slotted(*) {
-  font-family: inherit !important;
+  /* font-family: inherit !important; */
   line-height: inherit !important;
 }
 
 ::slotted(:is(h1, h2, h3, h4, h5, h6)) {
   font-size: inherit !important;
```

Comment:
Does the font size have to be inherited here? When I had to use a heading in the card body, I needed it to be a different size than the body copy.